File: 240719/평균점수 구하기/print-average.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        final int STUDENT_CNT = 8;
        double sum = 0;

        for(int i = 0; i < STUDENT_CNT; i++){
            sum += sc.nextDouble();
        }

        System.out.println(Math.round(sum / STUDENT_CNT * 10) / 10.0);

    }
}

File: 240719/평균점수 구하기/print-average.js
const fs = require('fs');
const scores = fs.readFileSync(0).toString().trim().split(' ').map(Number);

const STUDENT_CNT = 8;

const sum = scores.reduce((acc, cur)=>acc+cur, 0);

console.log(Math.round(sum / STUDENT_CNT * 10) / 10);